flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/MapView.java

```diff
@@ -255,26 +248,4 @@ public static DataType newMapViewDataType(DataType keyDataType, DataType valueDa
                 DataTypes.FIELD(
                         "map", DataTypes.MAP(keyDataType, valueDataType).bridgedTo(Map.class)));
     }
-
-    // --------------------------------------------------------------------------------------------
-    // Legacy
-    // --------------------------------------------------------------------------------------------
-
-    @Deprecated public transient TypeInformation<?> keyType;
-
-    @Deprecated public transient TypeInformation<?> valueType;
-
-    /**
-     * Creates a {@link MapView} with the specified key and value types.
-     *
-     * @param keyType The type of keys of the map view.
-     * @param valueType The type of values of the map view.
-     * @deprecated This method uses the old type system. Please use a {@link DataTypeHint} instead
-     *     if the reflective type extraction is not successful.
```

Comment:
Again I think its probably worth retaining the advice from this javadoc somewhere.

Reply:
Thanks for the review. What do you think of something like:
```
{@code MapView(TypeInformation<?> keyType, TypeInformation<?> valueType)} method was deprecated and removed. Please use a {@link DataTypeHint} instead.
```